config/default.json

```diff
@@ -89,5 +89,9 @@
         "port": 5052
       }
     }
+  },
+  "caching": {
+    "enabled": true,
+    "refreshMillis": 5000
```

Comment:
I found that 5s goes so quickly just playing around with it on my local system. I was even tempted to up this to 10s :) But sure, yeah, let's put this on 1s by default - minimize weird results for basic users.